packages/calypso-e2e/src/lib/pages/gutenberg-editor-page.ts

```
@@ -255,6 +248,32 @@ export class GutenbergEditorPage {
 		return lines.filter( Boolean ).join( '\n' );
 	}
 
+	/**
+	 * Closes all panels that can be opened in the editor.
+	 *
+	 * This method will attempt to close the following panels:
+	 * 	- Publish Panel (including pre-publish checklist)
+	 * 	- Editor Settings Panel
+	 * 	- Editor Navigation Sidebar
+	 */
+	async closeAllPanels(): Promise< void > {
+		try {
+			await this.editorPublishPanelComponent.closePanel();
+		} catch {
+			// noop
```

Should we just let this throw if it errors? All `close` methods are designed to be error safe, so if something _does_ error here, I figure it might actually be meaningful and worth bubbling up!

---

I've switched to use the `Promise.allSettled` construct so in that case, any uncaught error would be thrown, so this should be good?

---

Fixed in 781fadad87fb98107179c863cb5013ff141a6030.